### PR TITLE
Fixes bloodpack sprite issues

### LINKED ISSUE #35279: Blood packs have question marks
Issue reported from  Round ID: 83027 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])

Reporting client version: 511

l2lbemm 1

### PR DESCRIPTION
**Pull request by nicbn**
Fixes #35279

### CODE REVIEW

File: code/modules/reagents/reagent_containers/blood_pack.dm [hunk 1]
```diff
@@ -44,8 +44,9 @@
 	icon_state = "random_bloodpack"
 
 /obj/item/reagent_containers/blood/random/Initialize()
+	icon_state = "bloodpack"
 	blood_type = pick("A+", "A-", "B+", "B-", "O+", "O-", "L")
-	. = ..()
+	return ..()
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/blood_pack.dm]: that.... means the same thing.

Comment (nicbn) [reply code/modules/reagents/reagent_containers/blood_pack.dm]: Saves a buffer I think, improving performance by one operation

File: code/modules/reagents/reagent_containers/blood_pack.dm [hunk 2]
```diff
@@ -71,10 +72,6 @@
 /obj/item/reagent_containers/blood/universal
 	blood_type = "U"
 
-/obj/item/reagent_containers/blood/empty
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/blood_pack.dm]: ....why?

Comment (nicbn) [reply code/modules/reagents/reagent_containers/blood_pack.dm]: Same thing as `/obj/item/reagent_containers/blood`

**Pull request merged**